app/models/badminton.rb

```diff
@@ -0,0 +1,2 @@
+class Badminton < Coaching
```

Comment:
badminton should be a sport and not a coaching. coaching should be of a sport

Reply:
Yes sir, fixed.
now coachings belongs to sports